tests/std/tests/VSO_0000000_c_math_functions/test.compile.pass.cpp

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#if BOTH
+#include "both.cpp"
+#elif CMATH
+#include "cmath.cpp"
+#elif CSTDLIB
+#include "cstdlib.cpp"
+#endif

```

Comment:
Just in case someone in the future attempts to compile this by hand, I think we could add an `#error` explaining the expected usage.